src/vis/bar/BarChart.tsx

```diff
@@ -134,7 +135,7 @@ export function BarChart({
           ) : null}
         </Box> */}
 
-        <ScrollArea.Autosize mah={containerHeight} maw={containerWidth}>
+        <ScrollArea.Autosize h={containerHeight} style={{ overflowX: 'hidden' }}>
```

Comment:
ScrollArea already has a inbuilt way of specifying which scrollbars to show, also do we need an autosizing scrollarea when we do not set the maximum width/height?

```suggestion
        <ScrollArea.Autosize  h={containerHeight} w={containerWidth} scrollbars="y">
```

Reply:
This works as expected. Thanks for the suggestion!